recipe/meta.yaml

```diff
@@ -24,15 +24,14 @@ outputs:
     version: {{ version }}
     build:
       script: {{ PYTHON }} -m pip install --no-deps -vv source/flit_core/
-      noarch: python
```

Comment:
I have already specified skipping for py<36 above, as per source code developers requirements. Since our lowest supported py is 3.8 (defined in aggregate cbc), py<38 will be implicitly applied.